docs/extra/logs-collect.md

```diff
@@ -0,0 +1,23 @@
+# Collecting logs for Support
+
+To help troubleshoot issues, obtain the logs from your Codacy instance and send them to Codacy's Support:
+
+1.  Collect logs from your Codacy instance using Fluentd:
+
+    ```bash
+    --set fluentdoperator.enabled=true
+    ```
+
+    After executing this command the Fluentd daemonset will send the logs to MinIO, which is also installed by the Codacy chart.
+
+2.  Download the logs by running the following command locally, replacing `<namespace>` with the namespace in which Codacy was installed:
+
+    ```bash
+    bash <(curl -fsSL https://raw.githubusercontent.com/codacy/chart/master/docs/extra/extract-codacy-logs.sh) -n <namespace>
```

Comment:
The command now has another option I forgot to document. You can use the `-d <DAYS_NUM>` flag to extract just the last `<DAYS_NUM>` of logs. This flag is optional, defaults to 7 days and should have a value between 1 and 7. If you run the command without any arguments it will show you the usage, where this is documented.

Reply:
Yes, and since this is useful to reduce the size of the .zip file, I included this as a suggestion.